test/euphonic_horace_test.py

```diff
@@ -58,7 +58,14 @@ def get_expected_output_filename(material_name, pars, opts):
     if opts['conversion_mat'] is not None:
         fname += '_conv_mat_det' + str(np.linalg.det(opts['conversion_mat']))
     if opts['lim'] is not None:
-        fname += '_lim' + str(opts['lim'])
+        # Account for different str conversion of positive exponent
+        # sci notation - e.g. in Matlab string(1e2) = "100" but in
+        # Python str(1e2) = "100.0" unless explicitly cast to int
+        if opts['lim']%1 == 0:
+            strlim = str(int(opts['lim']))
+        else:
+            strlim = str(opts['lim'])
```

Comment:
OK it has dawned on me that in python % works on reals and will give the decimal part of opts here. But for a naive C++ programmer this is very counter-intuitive. If you had a comment after "==0:" like # no fractional part, and after "else:" like # positive fractional part, that could help some maintenance programmer in the future.

Reply:
Done